### ISSUE TITLE
Tag does not lock upstream OpenBSD sources

### ISSUE THREAD

**Issue by comutt** [libressl-portable/portable#110]
This repository libressl-portable/portable provides tags such as v2.2.1,
but because update.sh always pull from upstream libressl-portable/openbsd
so it seems that there is no way to determine/checkout upstream openbsd sources.

I think upstream openbsd repository should also provide tags of LibreSSL (not OpenBSD),
or on this repository tie upstream commit-ish with tags in some ways.

Getting tarball is not good in some situation; if someone want to use `git submodule` then some solutions other than tarball are hoped.

**Comment by busterb** [comment]
This has been mentioned a few times before. The upstream openbsd repository is maintained in CVS, so we don't have any upstream tags to work with (the CVS tags are reserved for OpenBSD releases).

We might be able to use a script like this to add tags post-conversion, to an extent:

```
for tag in `git tag`; do
        release_date=`git show -s --format=%ci $tag`
        commit=`git -C openbsd rev-list -n 1 --before="$release_date" master`
        git -C openbsd tag $tag $commit
        echo Tagged $tag as $commit in openbsd
done
```

**Comment by comutt** [comment]
@busterb 
Your suggestion seems good for me.
If you can do such tagging work from the next release, feel free to close this issue.

**Comment by busterb** [comment]
Tags are now included in the openbsd tree under the prefix libressl-vX.Y.Z. The trees do not automatically lock, but I included a script that can regenerate older releases: https://github.com/libressl-portable/portable/blob/master/check-release.sh

This script revealed a few minor tag problems, namely that 'mdc2.h' is erroneously included with the 2.2.0-3 releases, and there is a manpage update is missing from 2.2.2. This script an the new tags will be helpful for verifying releases in the future.
